source/puppet/4.0/reference/upgrade_agent.markdown

```diff
@@ -23,17 +23,17 @@ upgrade instructions](server_upgrade.markdown) on at least one host so the agent
 * Copy your SSL certificate tree from its previous location (from `puppet agent --configprint ssldir` above) to its new 
   AIO path: `/etc/puppetlabs/puppet/ssl`, making sure to preserve permissions and ownership. For example:
 
-    `cp -rp /var/lib/puppet/ssl /etc/puppetlabs/puppet/ssl`
+`cp -rp /var/lib/puppet/ssl /etc/puppetlabs/puppet/ssl`
```

Comment:
This outdenting is wrong, because we want the example to stay associated with the list item. If we do this, it turns into a bunch of one-item lists with the code examples dissociated from them.